Team page component

vicmator opened the pull request.

Commit:
First step Driver Component

Commit:
Created new dialog to change Pilot

Commit:
Merge branch 'dev' into driverComponent

Commit:
DriverComponent with Menu

Commit:
Start with Teams page Component

danilupion committed:
Fix for main teams problems

Commit:
Show table mock

Commit:
Merge branch 'driverComponent' into teamPageComponent

danilupion committed:
Fixed issue with repeated imported module

Commit:
Team Page Table

Commit:
Driver Page

Commit:
Remove unnecessary method

Commit:
Merge branch 'dev' into teamPageComponent

Commit:
Change style pointsOptionDriver

Commit:
Change in styles

Commit:
Merge branch 'dev' into teamPageComponent

danilupion commented on src/app/components/driver/driver.component.html:
```diff
@@ -0,0 +1,18 @@
+      >more_vert</mat-icon>
+    </div>
+  </button>
+    <mat-menu #menuPilots="matMenu">
```
Wrong indentation

vicmator replied:
Done

danilupion commented on src/app/components/driver/changePilotDialog/changePilotDialog.component.ts:
```diff
@@ -0,0 +1,30 @@
+})
+export class ChangePilotDialogComponent {
+  @Output()
+  result = new EventEmitter();
```
(opt) You may realize that I trie to restrict as much as possible methods and properties and even for public stuff I explicitly flag them as public

vicmator replied:
You are right. Done

danilupion commented on src/app/components/driver/driver.component.scss:
```diff
@@ -0,0 +1,21 @@
+    width: 25px;
+    height: 25px;
+
+    /deep/.mat-button-wrapper {
```
WTF is this? 😮

vicmator replied:
Answered face-to-face. it will be removed in the future

danilupion commented on src/app/pages/main/drivers/drivers.page.html:
```diff
@@ -0,0 +1,28 @@
+<mat-card>
```
wrong indentation, please use two spaces

vicmator replied:
Done

danilupion commented on src/app/pages/main/teams/teams.page.html:
```diff
@@ -1,6 +1,39 @@
 <mat-card>
```
Indentation 😛

vicmator replied:
Done

Commit:
PR Inputs, relocated flags and change the way to hidden button driver when logout
